.github/workflows/integrationTest.yml

```diff
@@ -0,0 +1,195 @@
+name: Deploy app to ubuntu server
+
+on: push
+
+jobs:
+
+  Terraform:
+    name: 'Terraform'
+    runs-on: ubuntu-latest
```

Comment:
The reason why I chose to run on ubuntu over other linux distro is for github actions it seems ubuntu is the default linux. This will support more distro in the future such as al2.

Reply:
The runner can be forever ubuntu since we only care about the instances on terraform. By support al2 I mean add al2 to the terraform list to run on.